engine/src/pausables.rs

```diff
@@ -116,17 +116,13 @@ impl<I: IO> EnginePrecompilesPauser<I> {
         match self.io.read_storage(&Self::storage_key()) {
             None => PrecompileFlags::empty(),
             Some(bytes) => {
-                let int_length = core::mem::size_of::<u32>();
-                let input = bytes.to_vec();
+                const U32_SIZE: usize = core::mem::size_of::<u32>();
+                assert_eq!(bytes.len(), U32_SIZE, "ERR_CORRUPTED_PAUSE_FLAGS");
```

Comment:
We need get out of the habit of having `ERR_*`, instead using it to be a bit more like actual errors. There should be an overhaul later, given that internal services also use the engine. A bit like this, for now, is ok: https://github.com/aurora-is-near/aurora-engine/blob/feat/erc20-gas-token/engine/src/state.rs#L59-L84.

Reply:
@joshuajbouw, your proposal is overkill, in this case, IMO. 
1. Because it's a panic and not returning an error.
2. In this module, we don't use errors at all. And therefore creating an additional error module is overkill.

Reply:
Ah fair, in this regard. I just see `"ERR_*"` and get triggered.